SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,8 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||wf.typesquare.com^$~font
```

Comment:
Not sure about this rule. I see only fonts on wf.typesquare.com

Reply:
d15fc750822137058677b28f9e710cdb9cadb425